Makefile

```diff
@@ -74,3 +74,6 @@ site-check: ## Test site for broken catalog links.
 update-function-docs: ## Update documentation for a function release branch
 	(cd scripts/update_function_docs/ && go build -o update_function_docs .)
 	RELEASE_BRANCH=$(RELEASE_BRANCH) ./scripts/update_function_docs/update_function_docs
+
+license:
+	cd functions/go && $(MAKE) license
```

Comment:
Have you tried this? running `make license` under functions/go works well, but it seems that this rule does not actually update the license.

Reply:
It seems you only add it for golang part. Root level makefile should be able to handle all golang/ts functions in curated (functions/) and contrib (contrib/functions/).